.github/workflows/testpythonpackage.yml

```diff
@@ -0,0 +1,55 @@
+# SPDX-FileCopyrightText: 2021-2024 The Ikarus Developers mueller@ibb.uni-stuttgart.de
+# SPDX-License-Identifier: CC0-1.0
+
+name: Test Python Package
+on:
+  push:
+    paths-ignore:
+      - 'docs/**'
+      - '.github/workflows/ghpages.yml'
+      - '.github/workflows/createDockerContainer.yml'
+      - '**.md'
+
+  pull_request:
+    types: [opened]
+    branches:
+      - main
+    paths-ignore:
+      - 'docs/**'
+      - '.github/workflows/ghpages.yml'
+      - '.github/workflows/createDockerContainer.yml'
+      - '**.md'
+
+jobs:
+  build-package:
+    runs-on: ubuntu-latest
+    container:
+      image: ikarusproject/ikarus-dev:latest
+      options: --memory-swap="20g" --memory="20g" --cpus="2" --user root
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install twine and scikit-build
+        run: |
+          /dune/dune-common/build-cmake/run-in-dune-env pip install twine scikit-build
+          git config --global --add safe.directory /__w/ikarus/ikarus
+
+      - name: Create tarball
+        run: |
+          /dune/dune-common/build-cmake/run-in-dune-env python setup.py sdist
+      
+      - name: Get package version
+        id: get_version
+        run: |
+          echo "version=$(grep -oP 'ikarusVersion = "\K[^"]+' setup.py)" >> "$GITHUB_ENV"
+          
+      - name: Install package from tarball
+        run: |
+          /dune/dune-common/build-cmake/run-in-dune-env pip install dist/pyikarus-$version.tar.gz --verbose --upgrade --no-build-isolation
+      
+      - name: Run Tests
+        run: |
+          /dune/dune-common/build-cmake/run-in-dune-env python ikarus/python/test/linearelastictest.py
```

Comment:
Maybe you can go through every test and just remove the import setpath statements. We can fix this later, when clion is ready to real docker container development

Reply:
You can also completley remove the set path workaround

Reply:
For this you have to remove: https://github.com/ikarus-project/ikarus/blob/10835528b8a6929962d14e3ab90d0d3728e69664/ikarus/python/test/CMakeLists.txt#L4 This configure line and remove the correspding file. Can you the also rename this PR accordingly?